app/views/settings/edit.html.erb

```diff
@@ -1,5 +1,5 @@
 <%= form_for @setting, url: '/settings', method: :patch do |f| %>
   <%= f.label :auto_text %>
   <%= f.text_field :auto_text, class: "form-control form-control-warning" %>
-  <%= f.submit %>  
-<% end %>
\ No newline at end of file
+  <input type="submit" value="Update" class="btn btn-primary">
```

Comment:
@xthegoodguy Why did we change the button? Was it just to add the class? You could just do this: `<%= f.submit 'Update', class: 'btn btn-primary' %>`